.github/workflows/main.yml

```diff
@@ -51,24 +51,41 @@ jobs:
                # Strip "v" prefix from tag name
                [[ "${{ github.ref }}" == "refs/tags/"* ]] && IMAGE_VERSION=$(echo $IMAGE_VERSION | sed -e 's/^v//')
                echo "IMAGE_VERSION=$IMAGE_VERSION" >> $GITHUB_ENV
-
+         
                echo "IMAGE_ID=$(echo ghcr.io/${{ github.repository_owner }}/$IMAGE_NAME | tr '[A-Z]' '[a-z]')" >> $GITHUB_ENV
 
-         # region See: https://docs.github.com/en/packages/managing-github-packages-using-github-actions-workflows/publishing-and-installing-a-package-with-github-actions#upgrading-a-workflow-that-accesses-ghcrio
-         -  name: Build Image
-            run: |
-               docker build -t $IMAGE_NAME .
+         # region See: https://docs.github.com/en/packages/managing-github-packages-using-github-actions-workflows/publishing-and-installing-a-package-with-github-actions#upgrading-a-workflow-that-accesses-ghcrio     
+         -  name: Use QEMU
+            uses: docker/setup-qemu-action@master
+            with:
+             platforms: all
```

Comment:
should this match the `platforms: linux/arm64,linux/amd64` below?

Reply:
You are correct.